components/suggest/src/schema.rs

```diff
@@ -127,7 +127,7 @@ pub const SQL: &str = "
     -- Just store the MD5 hash of the dismissed suggestion.  The collision rate is low and the
     -- impact of a collision is not showing a suggestion, which is not that bad.
     CREATE TABLE dismissed_suggestions (
-        url_hash INTEGER PRIMARY KEY
+        url TEXT PRIMARY KEY
```

Comment:
Now that we're storing raw URLs, let's the drop the above comment about hashing.